src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -187,6 +187,14 @@ def register_options(cls, register):
     register('--suggest-missing-deps', type=bool,
              help='Suggest missing dependencies on a best-effort basis from target\'s transitive'
                   'deps for compilation failures that are due to class not found.')
+    
+    register('--missing-deps-not-found-msg', type=str,
```

Comment:
This one should be marked advanced, as users are unlikely to change it... only folks deploying pants.